kge/job/auto_search.py

```diff
@@ -49,24 +49,16 @@ def save(self, filename):
             filename,
         )
 
-    def resume(self, checkpoint_file=None):
-        if checkpoint_file is None:
-            last_checkpoint = self.config.last_checkpoint()
-            if last_checkpoint is not None:
-                checkpoint_file = self.config.checkpoint_file(last_checkpoint)
-
-        if checkpoint_file is not None:
-            self.resumed_from_job_id = self.load(checkpoint_file)
-            self.trace(
-                event="job_resumed", checkpoint_file=checkpoint_file
-            )
-            self.config.log(
-                "Resumed from {} of job {}".format(
-                    checkpoint_file, self.resumed_from_job_id
-                )
+    def load(self, checkpoint):
```

Comment:
We should try to use consistent names (load_from, save_to)

Reply:
instance methods now called load and static/class methods create_from